src/widgets/Menu/Menu.tsx

```diff
@@ -132,6 +139,7 @@ const Menu: React.FC<NavProps> = ({
           isDark={isDark}
           toggleTheme={toggleTheme}
           langs={langs}
+          toggleNotPossible={toggleNotPossible !== undefined && true}
```

Comment:
Doesn't `toggleNotPossible={toggleNotPossible}` do the same?

Reply:
I was getting a typescript error and needed the !== undefined

Related to the intrinsic attributes piece